packages/keychain/src/components/layout/Container/Header/NetworkStatus.tsx

```diff
@@ -0,0 +1,51 @@
+import { StarknetColorIcon, StarknetIcon, SlotIcon } from "@cartridge/ui";
+import { QuestionIcon } from "@cartridge/ui/src/components/icons/utility/Question";
+import { Button, Circle } from "@chakra-ui/react";
+import { useConnection } from "hooks/connection";
+import { useToast } from "hooks/toast";
+import { constants } from "starknet";
+import { isSlotChain } from "utils/network";
+
+export function NetworkStatus() {
+  const { chainName, chainId } = useConnection();
+  const { toast } = useToast();
+
+  if (!chainName) {
+    return;
+  }
+
+  return (
+    <Button
+      size="sm"
+      bg="solid.primary"
+      fontSize="xs"
+      fontFamily="Inter"
+      _hover={{ bg: "solid.primary", opacity: 0.7 }}
+      iconSpacing={1.5}
+      leftIcon={
+        <Circle size={5} bg="solid.primary">
+          {(() => {
```

Comment:
Controller on localhost coming soon. I think we can first check rpcUrl and see if localhost then display Katana? Katana by default spins up with `SN_SEPOLIA`

Reply:
Oh good point, totally ! I'll add that